src/main/java/Storage/Storage.java

```diff
@@ -0,0 +1,124 @@
+package Storage;
+
+import TaskList.TaskList;
+import TaskList.Task;
+import TaskList.ToDos;
+import TaskList.Deadlines;
+import TaskList.Events;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import java.util.List;
+import java.text.DateFormat;
+import java.text.ParseException;
+
+public class Storage {
+
+    public final String FILEPATH;
+
+    public Storage(String filepath) {
+        this.FILEPATH = filepath;
+    }
+
+    /**
+     * store text into a file
+     * @param filePath
+     * @param textToAdd
+     * @throws IOException
+     */
+    public void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter((filePath));
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+    /**
+     * append the text in a file without over-writing the content inside
+     * @param filePath
+     * @param textToAppend
+     * @throws IOException
+     */
+    public void appendToFile(String filePath, String textToAppend) throws IOException {
```

Comment:
This method seems to be a duplicate with `writeToFile()`. You should abstract out the common code and keep these methods as wrapper functions, only adjusting the append=True.